Build failure with mtl-2.3

```
Building library for stache-2.3.1..
[1 of 6] Compiling Text.Mustache.Type ( Text/Mustache/Type.hs, dist/build/Text/Mustache/Type.o, dist/build/Text/Mustache/Type.dyn_o )
[2 of 6] Compiling Text.Mustache.Render ( Text/Mustache/Render.hs, dist/build/Text/Mustache/Render.o, dist/build/Text/Mustache/Render.dyn_o )

Text/Mustache/Render.hs:89:5: error:
    Variable not in scope: unless :: Bool -> Render () -> Render ()
   |
89 |     unless (isBlank val) $
   |     ^^^^^^

Text/Mustache/Render.hs:92:11: error:
    • Variable not in scope:
        forM_ :: [Value] -> (Value -> Render ()) -> Render ()
    • Perhaps you meant one of these:
        ‘V.forM_’ (imported from Data.Vector),
        ‘V.forM’ (imported from Data.Vector),
        ‘V.iforM_’ (imported from Data.Vector)
      Perhaps you want to add ‘forM_’ to the import list in the import of
      ‘Data.Foldable’ (Text/Mustache/Render.hs:26:1-27).
   |
92 |           forM_ (V.toList xs) $ \x ->
   |           ^^^^^

Text/Mustache/Render.hs:98:3: error:
    Variable not in scope:
      when :: Bool -> Render () -> ReaderT RenderContext (State S) ()
   |
98 |   when (isBlank val) $
   |   ^^^^
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+## Stache 2.3.2
+
+* Builds with `mtl-2.3`.
+
 ## Stache 2.3.1
 
 * Uses `aeson-2.x.x.x`.

--- a/Text/Mustache/Compile.hs
+++ b/Text/Mustache/Compile.hs
@@ -22,7 +22,8 @@ module Text.Mustache.Compile
 where
 
 import Control.Exception
-import Control.Monad.Except
+import Control.Monad (filterM, foldM)
+import Control.Monad.IO.Class (MonadIO (..))
 import qualified Data.Map as M
 import Data.Text (Text)
 import qualified Data.Text as T

--- a/Text/Mustache/Render.hs
+++ b/Text/Mustache/Render.hs
@@ -18,7 +18,8 @@ module Text.Mustache.Render
   )
 where
 
-import Control.Monad.Reader
+import Control.Monad (forM_, unless, when)
+import Control.Monad.Reader (MonadReader (local), ReaderT (runReaderT), asks)
 import Control.Monad.State.Strict (State, execState, modify')
 import Data.Aeson hiding (Key)
 import qualified Data.Aeson.Key as Aeson.Key

--- a/stache.cabal
+++ b/stache.cabal
@@ -89,17 +89,17 @@ executable stache
         ghc-options: -O2 -Wall
 
 test-suite tests
-    type:             exitcode-stdio-1.0
-    main-is:          Spec.hs
-    build-tools:      hspec-discover >=2.0 && <3.0
-    hs-source-dirs:   tests
+    type:               exitcode-stdio-1.0
+    main-is:            Spec.hs
+    build-tool-depends: hspec-discover:hspec-discover >=2.0 && <3.0
+    hs-source-dirs:     tests
     other-modules:
         Text.Mustache.Compile.THSpec
         Text.Mustache.ParserSpec
         Text.Mustache.RenderSpec
         Text.Mustache.TypeSpec
 
-    default-language: Haskell2010
+    default-language:   Haskell2010
     build-depends:
         aeson >=2 && <3,
         base >=4.13 && <5.0,

